src/commands/tag-repo.yml

```diff
@@ -2,7 +2,22 @@ description: >
   This command will tag the github repository with the semantic version of the project.
   This command should be used prior to deploying the project.
 
+parameters:
+  tag:
+    type: string
+    description: |
+      Tag being assigned
+
 steps:
   - run:
-      name: "Not Yet Implemented"
-      command: echo "Not Yet Implemented"
+      name: Validate params
+      command: >
+        if [[ -z "<<parameters.tag>>"]]; then
+          echo "tag parameter must be defined"
+          exit 1
+        fi
+  - run:
+      name: Tag Repo
```

Comment:
```suggestion
      name: "Tag $CIRCLE_PROJECT_REPONAME Repository"
```

Reply:
https://circleci.com/docs/2.0/env-vars/?utm_source=google&utm_medium=sem&utm_campaign=sem-google-dg--uscan-en-dsa-maxConv-auth-brand&utm_term=g_-_c__dsa_&utm_content=&gclid=Cj0KCQjwr-SSBhC9ARIsANhzu14qQHzsiV1UDPIUlBqyEjQUJqF-y_q74BFkorF2NMYyJ8SI6lYLBR4aAvgoEALw_wcB#built-in-environment-variables